package.json

```diff
@@ -405,6 +405,7 @@
     "@storybook/storybook-deployer": "^2.8.16",
     "@storybook/test-runner": "^0.10.0",
     "@storybook/theming": "^7.0.11",
+    "@testing-library/dom": "^9.3.1",
```

Comment:
I don't think we need to import this, we have used it just for within in the test. RTL already supports it. Can we remove it?